spec/unit/outputs/elasticsearch_secure_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "spec/es_spec_helper"
```

Comment:
Might be nice to name this spec elasticsearch_ssl.rb , a little more direct than 'secure', but not a big deal.